.github/workflows/gradle.yml

```diff
@@ -23,12 +18,24 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
+    
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    
     - name: Set up JDK 18
```

Comment:
Mod runs on J17, elaborate if this is fine or not

Reply:
it compiles fine, but it would probably be better to use J17